Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+WORKDIR /code
+
+COPY . /code
+RUN dotnet restore
+RUN dotnet build -c Release
+
+FROM mcr.microsoft.com/dotnet/runtime:5.0
```

Comment:
The runtime is way smaller than the whole SDK which results in a smaller docker container.